src/index.tsx

```diff
@@ -6,6 +6,8 @@ import './index.scss';
 import App from './app/App';
 import { store } from './app/store';
 import reportWebVitals from './reportWebVitals';
+import { ThemeProvider } from '@mui/material';
+import { theme } from './theme';
```

Comment:
I'm not sure why this change is in this PR

Reply:
The original branch was rebased from the demo branch before these changes were merged into `main`, so now that they are another rebase fixes this issue.